src/frontend/apps/lti_site/package.json

```diff
@@ -118,7 +118,7 @@
     "core-js": "3.25.0",
     "dompurify": "2.4.0",
     "github-markdown-css": "5.1.0",
-    "grommet": "2.25.1",
+    "grommet": "*",
```

Comment:
Seeing only one `*` here seems ugly to me.
Can't we leave the version number here ?

Reply:
I have mix feelings about that, I don't really like the star aswell, yet I find it's a nice solution to be sure our dependendies have all the same version. Tell me if you really want to use the version number here